### PR TITLE
Pre approval details [2/2] 

### PR DESCRIPTION
**Pull request by KrekoG**
### Added
- Details page for pre-approval (#160763347)

[Delivers [#160763347](https://www.pivotaltracker.com/story/show/160763347)]

### CODE REVIEW

File: app/Basket/ApplicationInitialisation/ApplicationInitialisationHelper.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+     * @return CustomerIntelligenceEntity
+     */
+    public function createCustomerIntelligenceEntity(Request $request)
+    {
+        $leadScoreId = $request->get('lead_score_id');
+        $preApprovalId = $request->get('pre_approval_id');
+
+        return CustomerIntelligenceEntity::make([
+            'lead_score_id' => $leadScoreId ? (int) $leadScoreId : null,
+            'pre_approval_id' => $preApprovalId ? (int) $preApprovalId : null,
```
Comment (EvanBarbour3) [inline app/Basket/ApplicationInitialisation/ApplicationInitialisationHelper.php]: `'pre_approval_id' => $preApprovalId ?: null`

**Comment by KrekoG** [comment]
Changes were requested this is being thrown back.

**Comment by KrekoG** [comment]
@wnowicki @EvanBarbour3 Please re review as a substantial part of the task has been removed

**Comment by KrekoG** [comment]
Another bunch of changes

**Comment by KrekoG** [comment]
@wnowicki @EvanBarbour3 Please re review this as there were change due to business request.

**Pull request merged**